tests/test_admin.py

```diff
@@ -1,9 +1,10 @@
-from django.test import SimpleTestCase
 from django.contrib.admin.sites import AdminSite
+from django.test import SimpleTestCase
```

Comment:
maybe we can add isort mandatory for the build

Reply:
`isort` is run in CI